### ISSUE TITLE
[question] Is 'theme' attr supported for the cards?

### ISSUE THREAD

**Issue by dedepene** [AmoebeLabs/swiss-army-knife-card#176]
## Bug report notice
**Before** you file a bug, make sure that you did not make a configuration mistake. The Swiss Army Knife in its current state does not yet have full configuration sanity checks!

So make sure your problem is not related to that!
- [x ] Yes, I'm sure that it is not a configuration mistake

## Bug description
I'm trying to get the [example battery card](https://swiss-army-knife-card-manual.amoebelabs.com/design/functional-card-battery-sensor2-card/) to work with proper colors while I'm using a theme (NM-Gonsboro) different from the one it's designed for (Material 3 theme D06, TealBlue). Does the custom card take the "theme:" attribute at all so I can apply D06 just for this card? So far had no luck with this:

```
          - type: horizontal-stack
            cards:        
              - type: 'custom:swiss-army-knife-card'
                theme: m3-d06-tealblue
                entities:
                
                  - entity: sensor.galaxy_watch5_j73x_battery_level
                    name: 'Watch'
                
                layout:
                  template:
                    name: sak_layout_fce_sensor2
                    variables:
                      - sak_layout_sensor_name_above_state: false
                      - sak_layout_sensor_colorstops_template: colorstops_battery_level
                      - sak_layout_sensor_icon_style: colorstops
                      - sak_layout_sensor_segarc_style: colorstops
                      - sak_layout_sensor_segarc_scale_max: 100
```
### Desktop browser (please complete the following information):
 - OS: [e.g. Windows]
 - Version [e.g. 110]

### Companion App on Smartphone/Tablet (please complete the following information):
 - Device: [e.g. iPhone16]
 - OS: [e.g. iOS16]
 - Version [e.g. 16]

**Additional context**

## (Optional): Suggested Solution
Any suggested solution to this bug.

**Comment by AmoebeLabs** [comment]
No, at this moment you cannot specify a theme for a single card.

I would have to check how this should be done. I assume that other cards are able to use a specific theme? In other words, I could check that code how this is done?

**Comment by AmoebeLabs** [comment]
Closed with #218
